user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Feeds/Feeds_Link.md

```diff
@@ -0,0 +1,37 @@
+---
+uid: Feed_Link
+---
+
+# Feed linking
+
+From DataMiner 10.3.10 onwards, [Dashboards](xref:newR_D) and [Low-Code Apps](xref:Application_framework) supports the usage of linking feeds through a textual syntax in some of the features. It allows dynamically referencing feed values from a Low-Code App or dashboard in text. Check the documentation of a feature to know wether or not it supports this new syntax.
```

Comment:
Is 10.3.10 the correct version? The title of the PR refers to RN 37229, which is set to 10.3.11.

Reply:
Should indeed by 10.3.11